### PR TITLE
Update MyScale in VDB Table

### PR DESCRIPTION
**Pull request by lqhl**

### CODE REVIEW

File: docs/tools/vdb_table/data/myscale.json [hunk 1]
```diff
@@ -132,13 +132,13 @@
     "comment": ""
   },
   "sharding": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://clickhouse.com/docs/en/engines/table-engines/special/distributed",
```
Comment (svonava) [inline docs/tools/vdb_table/data/myscale.json:136]: Any link for this within your docs? Might be clearer to not send the reader to 3rd party documentation, even if you use the 3rd party under the hood.

If not, then at least let's explain this in the comment: field.

File: docs/tools/vdb_table/data/myscale.json [hunk 2]
```diff
@@ -132,13 +132,13 @@
...
     "comment": ""
   },
   "doc_size": {
     "bytes": 0,
-    "unlimited": false,
+    "unlimited": true,
```
Comment (svonava) [inline docs/tools/vdb_table/data/myscale.json:141]: Any mention of this in the docs? Would be useful to understand the tradeoffs involved for large docs, if there are any,.

File: docs/tools/vdb_table/data/myscale.json [hunk 3]
```diff
@@ -43,28 +43,28 @@
   },
   "geo_search": {
     "support": "full",
-    "source_url": "https://myscale.com/docs/en/functions/geo-functions/",
+    "source_url": "https://myscale.com/docs/en/functions/geospatial-functions/",
     "comment": "via https://myscale.com/docs/en/functions/geospatial-functions/"
```
Comment (svonava) [inline docs/tools/vdb_table/data/myscale.json:47]: Perhaps let's set this comment as empty as it doesn't add much?

**Review by svonava — COMMENTED** [review]
Thank you for updating the information, left a few comments/questions.

File: docs/tools/vdb_table/data/myscale.json (hunk 2 above)
Comment (lqhl) [reply docs/tools/vdb_table/data/myscale.json:141]: We don't have a doc for this, so I added a comment.

**Comment by lqhl** [comment]
Hello @svonava, I have made the updates based on your comments. Could you please review them again?

**Comment by dhruv-anand-aintech** [comment]
looks good! merging now

**Pull request merged**